swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self,condition=0.0):
+        super().__init__(category= 'Clothing', condition=condition)
+       
+
+    
+    def __str__(self): 
+        return "The finest clothing you could wear." 
+
+
```

Comment:
Notice that we don't need to define `condition_description`. Since we want the same implementation for this child class as for the parent class, by leaving it out of the class definition, we will inherit the method from the parent.